app/ts/components/subcomponents/PreactBlocky.tsx

```diff
@@ -1,15 +1,13 @@
 import { useEffect, useState } from 'preact/hooks'
 import { DataURLCache } from './DataURLCache.js'
 import { JSX } from 'preact/jsx-runtime'
+import { addressString } from '../../utils/bigint.js'
 const dataURLCache = new DataURLCache()
 
 interface BlockieProps {
-	seed: string,
-	size?: number,
+	address: bigint,
```

Comment:
Consider a signal here since you have to touch all of the call sites anyway.
```suggestion
	address: Signal<bigint>,
```
Same for all of the properties, but this one is the most likely to change.